### PR TITLE
CCPA compliance

### PR DESCRIPTION
**Pull request by curlyblueeagle**
Adapter now sends USP consent info during bid request and user sync phases.

### CODE REVIEW

File: modules/33acrossBidAdapter.js [hunk 1]
```diff
@@ -6,7 +6,9 @@ const BIDDER_CODE = '33across';
 const END_POINT = 'https://ssc.33across.com/api/v1/hb';
 const SYNC_ENDPOINT = 'https://ssc-cms.33across.com/ps/?m=xch&rt=html&ru=deb';
 
-const adapterState = {};
+const adapterState = {
```
Comment (curlyblueeagle) [inline modules/33acrossBidAdapter.js:9]: We need to really re-visit this. Saving the adapter state i.e. auction state forever is bad practice. It has led to unit tests being stateful. 

For context, we had introduced `adapterState` in order to save the siteID of ad units passed through Bid Request phase in order to obtain them during user sync phase. The latter does not provide access (directly) to ad units.

File: test/spec/modules/33acrossBidAdapter_spec.js [hunk 2]
```diff
@@ -200,6 +219,7 @@ describe('33acrossBidAdapter:', function () {
 
   afterEach(function() {
     sandbox.restore();
+    spec.adapterState.uniqueSiteIds = [];
```
Comment (curlyblueeagle) [inline test/spec/modules/33acrossBidAdapter_spec.js:222]: This is what I mean by the unit tests being stateful. I am forced to reset states between tests for each item in adapter spec.

File: modules/33acrossBidAdapter.js [hunk 3]
```diff
@@ -84,7 +86,7 @@ function _createServerRequest(bidRequest, gdprConsent = {}, pageUrl) {
   ttxRequest.imp = [];
   ttxRequest.imp[0] = {
     banner: {
-      format: sizes.map(size => Object.assign(size, {ext: {}}))
+      format: sizes.map(size => Object.assign(size, { ext: {} }))
```
Comment (terryc33x) [inline modules/33acrossBidAdapter.js:89]: do we still need the empty ext in here?

File: test/spec/modules/33acrossBidAdapter_spec.js [hunk 4]
```diff
@@ -665,6 +763,7 @@ describe('33acrossBidAdapter:', function () {
           transactionId: 't2'
         }
       ];
+      console.log(spec.adapterState);
```
Comment (terryc33x) [inline test/spec/modules/33acrossBidAdapter_spec.js:766]: should we remove the console.log

File: test/spec/modules/33acrossBidAdapter_spec.js (hunk 2 above)
Comment (terryc33x) [reply test/spec/modules/33acrossBidAdapter_spec.js:222]: is it possible to clear `adapterState.uniqueSiteIds` in the `getUserSyncs()` function? so we do not need to export it and reset it

Comment (curlyblueeagle) [reply test/spec/modules/33acrossBidAdapter_spec.js]: Done.

File: test/spec/modules/33acrossBidAdapter_spec.js (hunk 4 above)
Comment (curlyblueeagle) [reply test/spec/modules/33acrossBidAdapter_spec.js]: Done

**Pull request merged**